apps/api/src/care-activity/care-activity.service.ts

```diff
@@ -124,15 +126,26 @@ export class CareActivityService {
       if (query.sortBy === CareActivitiesCMSFindSortKeys.UPDATED_BY) {
         orderBy = 'ca_up.displayName';
       }
+      if (query.sortBy === CareActivitiesCMSFindSortKeys.CARE_SETTING_NAME) {
+        orderBy = 'ca_cl.displayName';
+      }
 
       queryBuilder.orderBy(orderBy, sortOrder as SortOrder); // add sort if requested, else default sort order applies as mentioned in the entity [displayOrder]
     }
 
-    // return the paginated response
-    return queryBuilder
-      .skip((query.page - 1) * query.pageSize)
-      .take(query.pageSize)
-      .getManyAndCount();
+    const all = await queryBuilder.getRawMany();
```

Comment:
it's brutal, but I guess that we won't have too many activities to handle without performance impact.